### ISSUE TITLE
Can't load the game 

### ISSUE THREAD

**Issue by nikiforr** [GastronomistGuild/ExpandedFoods#34]
Is the mod compatible with the new version? If so, I can't load the world (both single and multiplayer) at all with this mod installed.
Was playing on 1.19.0 pre8 version.
[client-debug.txt](https://github.com/l33tmaan/EFRecipes/files/13842431/client-debug.txt)
[server-debug.txt](https://github.com/l33tmaan/EFRecipes/files/13842432/server-debug.txt)
[server-event.txt](https://github.com/l33tmaan/EFRecipes/files/13842433/server-event.txt)
[server-main.txt](https://github.com/l33tmaan/EFRecipes/files/13842434/server-main.txt)

**Comment by l33tmaan** [comment]
It should be, I'm able to work in 1.19.4 with this version of Expanded Foods and the most recent version of A Culinary Artillery. Are you still having this problem?

**Comment by nikiforr** [comment]
Nope, it's happened long ago when both CA and EF weren't updated. Now everything is fine, thanks!
